+sig/+node/Signal.m

```diff
@@ -286,53 +286,65 @@
     end
     
     function fs = flattenStruct(this)
-      % Use a struct with signal fields as a blueprint to wire up signals
-      % as inputs to this target so that their values will set the field
-      % values directly in the target's struct value all done in mexnet
-      % according to the transfer opcode
+      % Returns a signal `fs` whose value is a struct containing 
+      % "flattened" fields for the signal fields in the struct that 
+      % `this` has as its value. 
+      %
+      % Uses a signal (whose value is a struct with signal fields) as a 
+      % blueprint to wire up signals as inputs to this target. The values
+      % of these signal fields will be directly set as the target signal's
+      % struct field values. This is done in mexnet according to the 
+      % transfer opCode.
+      
       fs = applyTransferFun(this, 'sig.transfer.flattenStruct', [], '%s.flattenStruct()');
-%       state = StructRef;
-%       state.unappliedInputChanges = false;
-%       state.inputsToSubsref = containers.Map(...
-%         'KeyType', 'uint64', 'ValueType', 'any');
-%       fs = applyTransferFun(this, 'sig.transfer.flattenStruct', state,...
-%         '%s.flattenStruct()');
+
     end
 
     function fs = flatten(this)
-      % all done in mexnet according to the transfer opcode
```

Comment:
Is this true?

Reply:
this comment is not true, which is why I removed it

Reply:
what you are unsure about is indeed the case here